axon-server-connector/src/main/java/org/axonframework/axonserver/connector/event/axon/PersistentStreamMessageSource.java

```diff
@@ -39,7 +40,9 @@ public class PersistentStreamMessageSource implements SubscribableMessageSource<
     /**
      * Instantiates a {@code PersistentStreamMessageSource}.
      *
-     * @param name                       The name of the event processor.
+     * @param name                       The name of the persistent stream. It's a unique identifier of the
```

Comment:
This information is so important that I think it should also reside in the class-level JavaDoc. Just a sentence or two should suffice.

Reply:
This piece of documentation is still not present on the `PersistentStreamMessageSource` its class-level JavaDoc. To be concrete, somewhere on line 32 of this file.

Reply:
I know, I didn't resolve the comment nor reply, so it's still pending. I just wanted to know if I'm going in right direction :)